docs/source/tutorial.rst

```diff
@@ -22,11 +22,18 @@ your Pyramid config. For that you may use:
         config = Configurator()
         config.include('cornice')
         config.include('cornice_swagger')
+        config.register_swagger_ui(
+            title='MyAPI',
+            description="OpenAPI documentation",
+            version='1.0.0')
 
 
 If you don't know what this is about or need more information, please check the
 `Pyramid documentation <http://docs.pylonsproject.org/projects/pyramid>`_
 
+By default API explorer will be served under `/api-explorer` path in your
+application. You can easily configure the paths, required pyramid view
+permissions or route factory.
```

Comment:
I think it feels more natural to read *configure the paths, required permissions and Pyramid route factory*

Reply:
I think it's worth mention that additional kwargs should match the the `generate()` arguments.

Reply:
Thanks! Will fix that.